packages/@dcl/dcl-rollup/configs.ts

```diff
@@ -65,16 +65,26 @@ export function createEcsConfig(_options: { PROD: boolean }): RollupOptions {
   }
 }
 
-export function createSceneConfig(options: { PROD: boolean }): RollupOptions {
+export function createSceneConfig(options: {
+  PROD: boolean
+  single?: string
+}): RollupOptions {
   const sceneJsonPath = sys.resolvePath('./scene.json')
   const sceneJson = JSON.parse(sys.readFile(sceneJsonPath)!)
 
   console.assert(sceneJson.main, 'scene.json .main must be present')
-  console.assert(sceneJson.ecs7, 'scene.json `"ecs7": "true"` must be present')
+  console.assert(
+    sceneJson.runtimeVersion === '7',
```

Comment:
```suggestion
    sceneJson.runtimeVersion.startsWith('7'),
```

Reply:
this version of the build-ecs will only support `runtimeVersion: "7"`, we can use startsWith in the runtime of the kernel